postgres.go

```diff
@@ -612,18 +613,20 @@ type PostgresCreateOptions struct {
 
 	Fork *DatabaseFork `json:"fork,omitempty"`
 
-	EngineConfig *PostgresDatabaseEngineConfig `json:"engine_config,omitempty"`
+	EngineConfig   *PostgresDatabaseEngineConfig `json:"engine_config,omitempty"`
+	PrivateNetwork *DatabasePrivateNetwork       `json:"private_network,omitempty"`
```

Comment:
Normally I'd be a concerned about whether this type of substruct would diverge per-database type in the future, but I don't think that's likely in this case.